src/blockchain/shelley/utils.js

```diff
@@ -139,21 +180,32 @@ const fragmentToObj = (fragment: any, networkDiscrimination: number, extraData:
           type: CERT_TYPE.PoolRegistration,
           pool_id: poolId.to_string(),
           // we should be able to do this considering js max int would be 285,616,414 years
-          start_validity: parseInt(startValidity.to_string(), 10),
-          owners: pool_owners,
+          start_validity: parseInt(reg.start_validity().to_string(), 10),
```

Comment:
I think that this was a merge error? `start_validity` returns a wasm bound struct, causing a leak here.

Reply:
Yep, fixed to call the function